hopr/hopr-lib/src/lib.rs

```diff
@@ -888,7 +781,7 @@ impl Hopr {
                 .announce(&multiaddresses_to_announce, &self.me)
                 .await
             {
-                Ok(_) => info!("Announcing node on chain: {:?}", multiaddresses_to_announce),
+                Ok(_) => info!("Announced node on chain: {:?}", multiaddresses_to_announce),
```

Comment:
The past tense is incorrect here, since the announcement result is not awaited (would require a second await).